src/classes/Contest.ts

```diff
@@ -75,6 +78,14 @@ export class Contest {
     }
   }
 
+  setContestId(aString: string) {
+    try {
+      this.contestId = jsonQuery(this.contestIdQuery, { data: aString }).value;
+    } catch (e) {
+      console.log('Error:', e);
```

Comment:
_question_ under what circumstances would this error out? Why ignore the error instead of handling it somehow?